app/components/buttons_component.html.erb

```diff
@@ -1,4 +1,11 @@
-<div class="row justify-content-end">
+<div class="row justify-content-between">
+  <div class="col-auto">
+    <% if model_type == 'Work' %>
```

Comment:
Can you move this into `ButtonsComponent` (the `.rb` file)?

Reply:
Not sure what I should move (and same comment for the delete component)?  Can you render HTML from within the component class?  Or just create a boolean method instead of the comparison here?